src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -2,10 +2,6 @@
 
 import java.util.List;
 
-public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
+public interface MachineProducer<T> {
+    List<T> get();
 }
```

Comment:
Why we can do this: SomeProducer implements MachineProducer<Cat>
Let's limit the parmetricity